src/main/resources/kami_at.cfg

```diff
@@ -65,8 +65,14 @@ public net.minecraft.tileentity.TileEntity field_145854_h # blockType
 public net.minecraft.item.ItemTool field_77865_bY # attackDamage
 public net.minecraft.util.Timer * # All fields
 public net.minecraft.client.shader.ShaderGroup field_148031_d # listShaders
+public net.minecraft.client.gui.MapItemRenderer * # All fields
+public net.minecraft.client.gui.MapItemRenderer *() # All methods
+public net.minecraft.client.gui.MapItemRenderer$Instance *() # All methods
+public net.minecraft.client.gui.MapItemRenderer$Instance
 
 #Specific targeting past here
+
+public-f net.minecraft.client.gui.MapItemRenderer$Instance field_148240_d # All fields
```

Comment:
This isn't all fields lol, it's one specific field. Put the mapped name in the comment

Reply:
Okay. I had to change it from all fields and forgot to change it back.